### PR TITLE
Use watchify2

### PR DESCRIPTION
**Pull request by zoubin**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,11 +34,11 @@
     "through2": "^2.0.0",
     "vinyl-fs": "^2.2.1",
     "vinyl-source-stream": "^1.1.0",
-    "watchify": "^3.4.0"
+    "watchify2": "0.0.1"
```
Comment (springuper) [inline package.json]: why not 0.1.0?

**Comment by JulesWang** [comment]
approved

File: package.json (hunk 1 above)
Comment (zoubin) [inline package.json]: watchify2 is just a temporary solution. It is expected to be deprecated soon by watchify.
Anyway, there is no solid reason why not 0.1.0

Comment (zoubin) [inline package.json]: Use 0.1.0 now

**Comment by springuper** [comment]
LGTM

**Pull request merged**